assets/src/components/routeLadder.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { useEffect } from "react"
+import { fetchTimepointsForRoute } from "../api"
+import { LoadableTimepoints, Route, Timepoint } from "../skate"
+import {
+  Dispatch,
+  setLoadingTimepointsForRoute,
+  setTimepointsForRoute,
+} from "../state"
+import Loading from "./loading"
+
+interface Props {
+  route: Route
+  timepoints: LoadableTimepoints | undefined
```

Comment:
Thoughts on moving this `|` to `skate.d` so that the `undefined`=not asked and `null`=loading concepts are documented in one place?

One way to do that might be to change `LoadableTimepoints` to `Timepoint[] | null | undefined`, and then `TimepointsByRouteId` use `Timepoint[] | null` instead of `LoadableTimepoints`